data/death_match_locations.json

```diff
@@ -26,6 +26,34 @@
         ],
         "boundaries": [ 2110.8208, 2055.1360, 2200.4045, 2144.6980 ]
     },
+    {
+        "id": 6,
+        "name": "LV RW FightClub (lag shot)",
```

Comment:
LV fightclub isn't really a popular fight zone for RW. (Due to it being very small, so only works for a small amount of players)
Maybe the slot is better used @ SF Military? As that one is quite often used.

Reply:
Good call. Updated.